tests/unit/test_main.py

```diff
@@ -53,13 +57,17 @@ def test_add_pipeline_error_empty_content(self):
 
     def test_list_pipelines(self):
         """Test list pipelines"""
+        self.napp.pipeline_controller.get_pipelines.return_value = {}
```

Comment:
This return_value isn't representative of what the mocked method should be returning, could you include a more closely expected response `{"pipelines": []}`, with `"pipelines"` key in it?